platform/linux-generic/pktio/stats/ethtool_stats.c

```diff
@@ -131,9 +131,15 @@ static int ethtool_stats(int fd, struct ifreq *ifr, odp_pktio_stats_t *stats)
 		} else if (!strcmp(cnt, "rx_packets")) {
 			stats->in_packets = val;
 			cnts++;
-		} else if (!strcmp(cnt, "rx_ucast_packets")) {
+		} else if (!strcmp(cnt, "rx_unicast_packets")) {
```

Comment:
Extra stats PR touches this same line. Better leave it as is here to avoid merge conflict.

Reply:
Fixed in V7.